controllers/common/hash.go

```diff
@@ -0,0 +1,18 @@
+package common
+
+import (
+	"hash/fnv"
+)
+
+/*
+	fnv64a is a 64-bit non-cryptographic hash algorithm with a low collision and a high distribution rate.
+	https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function
+*/
+func FnvHash(s string) uint64 {
+	h := fnv.New64a()
```

Comment:
use 32 to keep the length minimal? and add a test case for this one

Reply:
Makes sense to keep the length to minimum but there will be more room for collisions. I am changing it to 32 for time being